formula/src/test/java/com/instacart/formula/subjects/EventCallbackFormula.kt

```diff
@@ -17,7 +17,7 @@ class EventCallbackFormula : Formula<Unit, String, EventCallbackFormula.Output>
         return Evaluation(
             output = Output(
                 state = state,
-                changeState = context.onEvent { newState ->
+                changeState = context.onEvent<String> { newState ->
```

Comment:
This actually adds up to the code readability.

Reply:
Do you mean improves code readability?